src/components/Map/Map.js

```diff
@@ -69,8 +69,10 @@ const Map: ComponentType<Props> = ({
   useEffect(() => {
     const initializeMap = () => {
       const map = L.map('map', {
-        center: [55, -4],
-        zoom: 4.5, 
+        center: [55.7, -3.7],
+        zoom: 5.4,
+        minZoom: 5.4,
+        maxZoom: 11,
```

Comment:
@annapowellsmith Could we increase maxZoom to 12 please? For areas like London with small UTLAs it helps to zoom in a little more.